src/measure_theory/measure/finite_measure_weak_convergence.lean

```diff
@@ -0,0 +1,276 @@
+/-
+Copyright (c) 2021 Kalle Kytölä. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kalle Kytölä
+-/
+import measure_theory.measure.measure_space
+
+/-!
+# Weak convergence of (finite) measures
+
+This file will define the topology of weak convergence of finite measures and probability measures
+on topological spaces. The topology of weak convergence is the coarsest topology w.r.t. which
+for every bounded continuous `ℝ≥0`-valued function `f` on, the integration of `f` against the
+measure is continuous.
+
+TODOs:
+* Rename the types `finite_measures α` -> `finite_measure α` and
+  `probability_measures α` -> `probability_measure α` after PR #8831.
+* Define the topologies (the current version only defines the types) via
+  `weak_dual ℝ≥0 (α →ᵇ ℝ≥0)`.
+* Prove that an equivalent definition of the topologies is obtained using bounded continuous
+  `ℝ`-valued functions instead.
+* Include the portmanteau theorem on characterizations of weak convergence of (Borel) probability
+  measures.
+
+## Main definitions
+
+The main definitions are the types `finite_measures α` and `probability_measures α`.
+
+TODO:
+* Define the topologies on the above types.
+
+## Main results
+
+None yet.
+
+TODO:
+* Portmanteau theorem.
+
+## Notations
+
+No new notation is introduced.
+
+## Implementation notes
+
+The topology of weak convergence of finite Borel measures will be defined using a mapping from
+`finite_measures α` to `weak_dual ℝ≥0 (α →ᵇ ℝ≥0)`, inheriting the topology from there.
+
+## References
+
+* [Billingsley, *Convergence of probability measures*][billingsley1999]
+
+## Tags
+
+weak convergence of measures, finite measure, probability measure
+
+-/
+
+noncomputable theory
+open measure_theory
+open set
+open filter
+open_locale topological_space ennreal nnreal
+
```

Comment:
Since the old `finite_measure` (now `is_finite_measure`) is in the `measure_theory` namespace, with full name `measure_theory.finite_measure`, I think it would be appropriate to do that for the new `finite_measure` too.  So can you please enclose the whole file in `namespace measure_theory .... end measure_theory`?

Reply:
Done!

(Although I will still test that the nested namespaces work as intended, i.e., we get e.g., `measure_theory.probability_measures.to_fun_univ` etc. If not, I'll see what is the right way.)